relayer/relays/beefy/polkadot-listener.go

```diff
@@ -41,7 +40,7 @@ func (li *PolkadotListener) Start(
 	}
 	li.beefyAuthoritiesKey = storageKey
 
-	requests := make(chan Request)
+	requests := make(chan Request, 1)
```

Comment:
Is the channel size 1 here nessessary?

Reply:
Setting the channel buffer size to 0 makes it a blocking channel, meaning that when `polkadot-listener` writes to the channel it will block until `ethereum-writer` has completed submitting the last commitment.

Setting the channel buffer size to 1 allows `polkadot-listener` to post a new commitment, and then immediately start looking for the next one without waiting for `ethereum-writer`.

If it takes `polkadot-listener` a long time to scan for the next commitment, this optimization could reduce the overall sync time.